src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1532,6 +1536,24 @@ protected override bool TryComputeLength(out long length)
             }
         }
 
+        [ConditionalTheory(nameof(IsWindows10Version1607OrGreater)), MemberData(nameof(Http2NoPushServers))]
```

Comment:
The existing test `SendAsync_RequestVersion20_ResponseVersion20IfHttp2Supported` already covers that case.  Basically, the expectation is that asking for HTTP/2 in the HttpRequestMessage.Version on systems that don't support it downgrade to HTTP/1.1.